code/upaya/app/assets/javascripts/homepage.js

```diff
@@ -0,0 +1,13 @@
+$(document).on('page:change', function () {
+  var maxH = 0;
+  $('.same-height-panel').each( function () {
+    var thisH = $(this).height();
+    if (thisH > maxH) {
+      maxH = thisH;
+    }
+  });
+
+  $('.same-height-panel').each( function () {
+    $(this).css('minHeight',maxH+'px');
+  });
+});
```

Comment:
> I actually thought about making this a jQuery plugin since I couldn't find anything pre-build online to do this.

let's do it (yet another side project :) )